mw-webapp/src/component/table/renderCellValue/renderICellsDayOff.tsx

```diff
@@ -0,0 +1,20 @@
+import {CellContext} from "@tanstack/react-table";
+import {DayReport} from "src/model/businessModel/DayReport";
+
+/**
+ * Render cell IsDayOff
+ * @param {CellContext<DayReport, boolean>} cellValue
+ * @returns {JSX.Element} JSX.Element
+ */
+export const renderCellIsDayOff = (cellValue: CellContext<DayReport, boolean>): JSX.Element => {
+  return (
+    cellValue.getValue() === true ?
+      <div>
```

Comment:
Only content changed, wrapper the same dry

Reply:
Don't understand what should I don't repeat(